src/components/subscriptions/data/contextHooks.jsx

```diff
@@ -11,9 +11,9 @@ export const useSubscriptionFromParams = ({ match }) => {
   // Use UUID to find matching subscription plan in SubscriptionContext, return 404 if not found
   const { params: { subscriptionUUID } } = match;
   const { data: subscriptions, loading } = useContext(SubscriptionContext);
-  const enterpriseSubscriptions = Object.values(subscriptions.results).filter(sub => sub.uuid === subscriptionUUID);
-  if (!subscriptions.count || enterpriseSubscriptions.length < 1) {
+  const foundSubscriptionsByUUID = Object.values(subscriptions.results).filter(sub => sub.uuid === subscriptionUUID);
```

Comment:
should we use .find() here instead of filter?

Reply:
Yeah, good suggestion! Will push up an update before merge :)